src/js/services/conversation-service.js

```diff
@@ -235,3 +246,28 @@ export function postPostback(actionId) {
         store.dispatch(showErrorNotification(store.getState().ui.text.actionPostbackError));
     });
 }
+
+
+export function fetchMoreMessages() {
+    const {conversation: {hasMoreMessages, messages}} = store.getState();
+
+    // TODO : add independent guard against parallel fetching
```

Comment:
Hmmm I guess I can just check the `appState` to see if `isFetchingMoreMessages` is true. Since we set it right before calling it.